packages/rest/src/rest.server.ts

```diff
@@ -151,7 +151,10 @@ export class RestServer extends Context implements Server, HttpServerLike {
   }
 
   get url(): string | undefined {
```

Comment:
Please add TSDoc comment to explain what is this property supposed to return. Make it clear that `basePath` is included in the URL.